src/logic/safe/store/middleware/safeStorage.ts

```diff
@@ -60,51 +54,13 @@ const safeStorageMware = (store) => (next) => async (action) => {
     const state = store.getState()
     const { dispatch } = store
     const safes = safesMapSelector(state)
-    const addressBook = addressBookSelector(state)
     await saveSafes(safes.toJSON())
 
     switch (action.type) {
       case ACTIVATE_TOKEN_FOR_ALL_SAFES: {
         recalculateActiveTokens(state)
         break
       }
-      case ADD_SAFE: {
```

Comment:
So the intention is to remove this ADD_SAFE --> ADD_OR_UPDATE_SAFE?

I still see many ADD_SAFE handlers but not many places where we call the addSafe action.
Is it worth to create a ticket to do this refactor in another PR?